### DISCUSSION TITLE
关于topk的设置问题

### DISCUSSION THREAD

**Discussion by ayicherry** [open-mmlab/mmclassification#1201] [Q&A] [answered]
在dataset的配置文件中有设置evaluation的topk，用来输出和作为best_model的选择依据，然后在model的配置参数中的head字段中也有个topk的设置，但是在代码端好像没有找到用的地方，不知道模型参数中的topk的作用是什么？

**Comment by yingfhu** [comment, marked as answer]
参考代码 [topk in head](https://github.com/open-mmlab/mmclassification/blob/master/mmcls/models/heads/cls_head.py#L41)，这里head里的topk用于在forward计算完loss之后计算当前batch的acc，需要使用`cal_acc`来控制。

**Reply by ayicherry** [reply]
明白了，看漏了代码，谢谢！
